hermes/src/api/types.rs

```diff
@@ -308,3 +309,20 @@ impl TryFrom<PriceUpdate> for PriceFeedsWithUpdateData {
         })
     }
 }
+
+#[derive(Debug, Clone, Serialize, Deserialize, ToSchema)]
+pub struct PriceFeedV2 {
```

Comment:
This name is a bit confusing, isn't it? I recommend `PriceMetadata`

Reply:
hmm we agreed on the design docs that price feeds would refer to the metadata itself and price updates would refer to the updates, maybe we can discuss more on this